AddonExample/AddonExample.csproj

```diff
@@ -9,8 +9,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>AddonExample</RootNamespace>
     <AssemblyName>AddonExample</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
```

Comment:
Can you please change this to `4.6.1`? I just changed this since apparently a lot of users have an older .NET installation and we don't really use anything that requires `4.7.1`.

Reply:
Changed by 296d77a